src/tea_html.resi

```diff
@@ -0,0 +1,203 @@
+module Cmds = Tea_html_cmds
+let map: ('a => 'b, Vdom.t<'a>) => Vdom.t<'b>
+let noNode: Vdom.t<'a>
+let text: string => Vdom.t<'a>
```

Comment:
Can you add some spacing to split them into rough sections? Use your judgment for what should be grouped together in the sections.

Reply:
And the same for the other files.